.azure-pipelines/unit_tests.yml

```diff
@@ -36,6 +36,9 @@ jobs:
       Pypy3:
         PYTHON_VERSION: "pypy3"
         TOXENV: "pypy3"
+      Lint:
+        PYTHON_VERSION: "3.5"
+        TOXENV: "flake8"
```

Comment:
Let's also remove it from the Py35 env then

Reply:
Done